scripts/scraper-ng/index.js

```diff
@@ -1,32 +1,45 @@
 const { RateLimit } = require("async-sema");
 const fetch = require("node-fetch");
+const fileReporter = require("vfile-reporter");
 const rehype = require("rehype");
-const reporter = require("vfile-reporter");
 
 const mdnUrl = require("./mdn-url");
+const summaryReporter = require("./vfile-reporter-summary");
 const toVFile = require("./url-to-vfile");
 
 const examplePage =
   "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div";
 const exampleShorthand = "/en-US/docs/Web/HTML/Element/div";
 
-const argv = require("yargs")
+const { argv } = require("yargs")
   .parserConfiguration({ "boolean-negation": false })
   .usage("Usage: $0 <url>")
   .example(`$0 ${examplePage}`, "scrape a page and all its subpages")
   .example(`$0 ${exampleShorthand}`, "omit the protocol and domain")
-  .example(`$0 --no-subpages ${exampleShorthand}`, "one page only")
+
+  .describe("n", "Dry run (lint-only)")
+  .alias("n", "dry-run")
   .example(`$0 -n ${exampleShorthand}`, "dry run (lint)")
-  .demandCommand(1, 1, "A URL is required", "Too many arguments")
+
   .describe("no-subpages", "Don't walk the entire tree from the URL")
   .default("no-subpages", false)
-  .describe("n", "Dry run (lint)")
-  .alias("n", "dry-run")
-  .describe("q", "Suppress success messages")
+  .example(`$0 --no-subpages ${exampleShorthand}`, "one page only")
```

Comment:
I was about to say this was completely unfixable, at least with yargs, because yargs doesn't give me any control over the layout of the examples. But I found that there's an option to let help expand to the full width of the terminal, which I've turned on—it's not a complete fix, but it does help.